WireAPI/Sources/WireAPISupport/Sourcery/AutoMockable.stencil

```diff
@@ -190,7 +190,7 @@ import {{ import }}
             {% call variableThrowableErrorUsage variable %}
             {% endif %}
             {% call mockedVariableName variable %}CallsCount += 1
-            if let {% call variableClosureName variable %} = {% call variableClosureName variable %} {
+            if let {% call variableClosureName variable %} {
```

Comment:
nice, we should make sure we use the same template across every sourcery invocations

Reply:
let's symlink all the stencils, no?

Reply:
Makes sense, could we do this in another PR?
The question remains where the original files are located then.

Reply:
sure let's do it in another one, I would even move the stencil file at the root directory or Wire-iOS